extensions/amp-analytics/0.1/requests.js

```diff
@@ -118,14 +118,16 @@ export class RequestHandler {
   send(configParams, trigger, expansionOption, dynamicBindings) {
     this.lastTrigger_ = trigger;
     const triggerParams = trigger['extraUrlParams'];
+    const filters = this.variableService_.getFilters();
+    const bindings = Object.assign({}, dynamicBindings, filters);
```

Comment:
Question: We never enable the filter feature before. But with your change, I would imaging that `TOUPPERCASE (${var})` would work. Is this OK to have w/o an experimental flag?

Reply:
Yes that would work. The issue is that if you want to nest these macros they need a different code path that does not (yet) have all the functionality like resolving syncronously

Reply:
Should we keep the `variable-filters` until we officially document the filter support? Or you think it's safe to remove the experimental flag now? 
I would imaging very few requests resolve to different value for example `TOUPPERCASE (${var})` we had here.

Reply:
protected bindings behind `v2` experiment flag